proto/internal.go

```diff
@@ -87,3 +91,190 @@ func InternalTimeSeriesDataFromValue(value *Value) (*InternalTimeSeriesData, err
 	}
 	return &ts, nil
 }
+
+// AccumulateFrom accumulates values from the supplied InternalTimeSeriesSample
+// into the existing values of the destination InternalTimeSeriesSample.
+func (dest *InternalTimeSeriesSample) AccumulateFrom(src *InternalTimeSeriesSample) {
```

Comment:
Any reason for this to be part of the `proto` package. If it is only used by `storage/engine/merge_test.go`, I'd prefer for it to be private to the `storage/engine` package.